src/misc/lv_style_gen.h

```diff
@@ -10,221 +10,221 @@
-extern const lv_style_prop_t _lv_style_const_prop_id_ANIM_TIME;
-void lv_style_set_transition(lv_style_t * style, const lv_style_transition_dsc_t * value);
-extern const lv_style_prop_t _lv_style_const_prop_id_TRANSITION;
-void lv_style_set_blend_mode(lv_style_t * style, lv_blend_mode_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_BLEND_MODE;
-void lv_style_set_layout(lv_style_t * style, uint16_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_LAYOUT;
-void lv_style_set_base_dir(lv_style_t * style, lv_base_dir_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_BASE_DIR;
-void lv_style_set_flex_flow(lv_style_t * style, lv_flex_flow_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_FLEX_FLOW;
-void lv_style_set_flex_main_place(lv_style_t * style, lv_flex_align_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_FLEX_MAIN_PLACE;
-void lv_style_set_flex_cross_place(lv_style_t * style, lv_flex_align_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_FLEX_CROSS_PLACE;
-void lv_style_set_flex_track_place(lv_style_t * style, lv_flex_align_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_FLEX_TRACK_PLACE;
-void lv_style_set_flex_grow(lv_style_t * style, uint8_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_FLEX_GROW;
-void lv_style_set_grid_column_dsc_array(lv_style_t * style, const int32_t * value);
-extern const lv_style_prop_t _lv_style_const_prop_id_GRID_COLUMN_DSC_ARRAY;
-void lv_style_set_grid_column_align(lv_style_t * style, lv_grid_align_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_GRID_COLUMN_ALIGN;
-void lv_style_set_grid_row_dsc_array(lv_style_t * style, const int32_t * value);
-extern const lv_style_prop_t _lv_style_const_prop_id_GRID_ROW_DSC_ARRAY;
-void lv_style_set_grid_row_align(lv_style_t * style, lv_grid_align_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_GRID_ROW_ALIGN;
-void lv_style_set_grid_cell_column_pos(lv_style_t * style, int32_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_GRID_CELL_COLUMN_POS;
-void lv_style_set_grid_cell_x_align(lv_style_t * style, lv_grid_align_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_GRID_CELL_X_ALIGN;
-void lv_style_set_grid_cell_column_span(lv_style_t * style, int32_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_GRID_CELL_COLUMN_SPAN;
-void lv_style_set_grid_cell_row_pos(lv_style_t * style, int32_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_GRID_CELL_ROW_POS;
-void lv_style_set_grid_cell_y_align(lv_style_t * style, lv_grid_align_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_GRID_CELL_Y_ALIGN;
-void lv_style_set_grid_cell_row_span(lv_style_t * style, int32_t value);
-extern const lv_style_prop_t _lv_style_const_prop_id_GRID_CELL_ROW_SPAN;
+LV_ATTRIBUTE_EXTERN_DATA extern const lv_style_prop_t _lv_style_const_prop_id_WIDTH;
```

Comment:
These are generated by [style_api_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/style_api_gen.py). Please update the script and run it again.

Reply:
did that.
I also tried to ran code-format.py, but this results in many FileNotFoundErrors. Can someone please run this (or tell me how to avoid these FileNotFoundErrors)
I ran `python code-format.py`

Reply:
I can't see that `[style_api_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/style_api_gen.py)` way updated. You should add `LV_ATTRIBUTE_EXTERN_DATA` in the script.

What is the exact log for `code-format.py`?